index.html

```diff
@@ -71,354 +70,122 @@ <h3>My Recent Works</h3>
                     <img id="img2" src="images/Img Placeholder (3).png" alt="">
                 </div>        
                 <div class="work-content">
-                    <h2>Multi-Post-Stories</h2>
+                    <h2>Multi-Post Stories</h2>
                     <p>
-                        A daily selection of privately personalized 
-                        <br> reads; no accounts or sign-ups required.
-                        <br> has been the industry's standard dummy
-                         text ever since the 1500s, when an 
-                        <br> unknown printer took a standard dummy
-                        <br> text.
+                        A daily selection
+                         of privately personalized reads; 
+                         no accounts or sign-ups <br> required. 
+                         This has been the industry's 
+                         standard dummy <br> text ever since 
+                         the 1500s, when an unknown
+                          printer took a <br>
+                         standard dummy text.
                     </p>
                     <div class="btn">
                         <ul>
-                            <li>CSS</li>
-                            <li>HTML</li>
-                            <li>Bootstrap</li>
-                            <li>Rubyy</li>
+                        <li>CSS</li>
+                        <li>HTML</li>
+                        <li>Boostrap</li>
+                        <li>Ruby</li>
                         </ul>
                     </div>
                     <div class="see_more_btn">
-                        <button class="view-project">See Projects</button>
+                        <button class="view-project" type="button">See Project</button>
                     </div> 
                 </div>
             </div>
```

Comment:
- [ ] Kindly remove this too. There's no exeptional. All the projects list should be populated from the array of objects from your js file. The same goes for the popup window for the project details according to the project requirements.

[OPTIONAL] Your popup window on the mobile does not look like is expected. Your project details card is expected to display all the information partaining to the project as demostrated on the Figma design. Kindly check the two screenshots below and walk your design alongside the Figma design. I marked this optional before it was not mentioned by the previous reviewer before me, but I would encourage your to give a serious attention.

### Your Design
![your-mobile-popup-window](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/26597432/17bd1c7f-4453-4c3f-a2a3-3816f4cf5383)

### Figma Design 
![temp-mobile-popup-window](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/26597432/8d4cf15b-3b97-46d0-bcaf-37bae05a79a7)